docs: update sandbox deployment docs

jonasdebeukelaer opened the pull request:
Add flytectl init instruction to the end of the sandbox deployment instructions to complete setup

evalsocket commented on rsts/deployment/sandbox.rst:
```diff
@@ -190,3 +190,7 @@ Flyte configuration on your remote cluster.
+
+#. In order to interact with your Flyte instance using ``flytectl``, initialise your configuration to point to this host ::
+
+    flytectl config init --host='http://localhost:30081'
```
make it `flytectl config init --host='localhost:30081'`

jonasdebeukelaer replied:
I'm embarrassed! Mindless copy-pasting the wrong command. fixing

evalsocket commented:
You can test your changes by running 
```bash
$ flytectl sandbox start  (In your case, You deployed it on aws and it will run the latest release in a docker for local development but both are sandbox)
# export KUBECONFIG=$KUBECONFIG:/Users/yuvraj/.kube/config:/Users/yuvraj/.flyte/k3s/k3s.yaml 
# Set context
# kubectl port-forward --address 0.0.0.0 svc/flyte-contour-envoy 30090:80 -n flyte
# flytectl config init --host='localhost:30081'
# flytectl get projects
```

evalsocket commented:
Please also sign the DCO https://github.com/flyteorg/flyte/pull/1947/checks?check_run_id=4521322174

jonasdebeukelaer commented on rsts/deployment/sandbox.rst:
```diff
@@ -190,3 +190,7 @@ Flyte configuration on your remote cluster.
     kubectl port-forward --address 0.0.0.0 svc/envoy 30081:80 -n projectcontour
```
the service `flyte-contour-envoy` is not part of the sandbox deployment (`deployment/sandbox/flyte_generated.yaml`) as far as I can see - is there a different method that should be used here?

evalsocket replied:
It is generated from `deployment/sandbox/flyte_generated.yaml`  [kustomize](https://github.com/flyteorg/flyte/tree/master/kustomize), But it's almost deprecated and people start using helm for there deployment and it is actively maintained  

cc: @EngHabu

jonasdebeukelaer replied:
makes sense, I'll redeploy using the helm chart then, and the first command will also need to be updated of it's outdated I imagine

evalsocket replied:
yes

jonasdebeukelaer commented:
Ok I've updated docs in two places as it seems a lot of things were quite outdated? Let me know your thoughts.

To make it work I also had to update the minio version in `/charts/flyte/values.yaml` from 
```
tag: 2021.10.13-debian-10-r0
```

to 
```
tag: 2021-12-10
```

as I was getting an image pullback error. Am I ok to add this in here too? I can rename to not just a  docs update accordingly.

evalsocket commented on charts/flyte/README.md:
```diff
@@ -1,59 +1,38 @@
-# flyte
+# Flyte Sandbox Chart
```
We don't edit readme directly, It will be generated by template that is available in same directory, After making your changes in template just run `make helm`

evalsocket commented:
I don't think so minio tag has any issue, Please add more details on that 
```
17:14:57 ➜ genesis-device docker pull ecr.flyte.org/bitnami/minio:2021.10.13-debian-10-r0
2021.10.13-debian-10-r0: Pulling from bitnami/minio
23b664af592e: Pull complete 
2b9568fe8603: Pull complete 
f7af824a81a6: Extracting [>                                                  ]   98.3kB/7.228MB
0a6aa41ece82: Download complete 
ab2dc04db2f9: Waiting 
4241bdcafa70: Waiting 
ced736c647a3: Waiting 
71c1458aff9f: Waiting 
703b638a8641: Waiting 
```

evalsocket commented:
@samhita-alla can you review the helm readme ? I think we need more changes , Actually that's our helm documentation that can be found here https://artifacthub.io/packages/helm/flyte/flyte 

We also need to add deployment documentation URL in helm chart readme, We can create a separate issue for that

jonasdebeukelaer commented:
Yep trying again the minio image is fine, not sure why I got those errors the first time.

I also found a bug in the readme generation, which I've fixed. It was causing `flyte` and `flyte-core` readmes to be identical.
Also moved my changes to the template.
